### DISCUSSION TITLE
adjust color on dark theme?

### DISCUSSION THREAD

**Discussion by realyukii** [mantinedev/mantine#4283] [Q&A] [answered]
does mantine support auto-detect color in dark mode, so i don't need to add ternary operator to determine which color to be picked?
i try to use theme.colors property, but the color still doesn't adjust

```html
<Text fz="sm" sx={(theme) => ({ color: theme.colors.gray })}>
Log in to view your "Followed" content.
</Text>
```

**Comment by rtivital** [comment, marked as answer]
No, you need to use `theme.colorScheme === 'dark' ? 'x' : 'y'` syntax to apply styles

**Reply by realyukii** [reply]
Very well done!

**Comment by twiddler** [comment]
I am facing a similar issue: I have a sticky container. When scrolling, the container does not have a background color, and as such, is transparent. I would like to set the background color to the background color of the current theme to solve this issue (although I am open to other ideas).

Using a ternary operator and picking the appropriate colors such as `theme.colors.gray[0]` by hand would cause issues when the background color changes one day. It'd be really cool to have something like `(theme) => theme.backgroundColor`. This way, any changes to theme itself could propagate throughout the app without any additional work.
